axum/src/routing/mod.rs

```diff
@@ -77,10 +77,7 @@ where
     }
 }
 
-impl<S, B> fmt::Debug for Router<S, B>
-where
-    S: fmt::Debug,
-{
+impl<S, B> fmt::Debug for Router<S, B> {
```

Comment:
Do we want to print the new `prev_route_id` field in the `Debug` impl?

Reply:
Sure! Added in 5c11c5c44d0e0